src/components/DayPickerNavigation.jsx

```diff
@@ -115,7 +115,9 @@ function DayPickerNavigation({
       )}
     >
       {!isVerticalScrollable && (
-        <button
+        <div
```

Comment:
i'm very confused why a "div that you're marking as being a button" is better than "a semantically correct button element" :-/

Reply:
@ljharb Did you read my description? The whole point is that with the way the custom button code is currently set up (the `navPrev`/`navNext` props), custom month navigation DOES NOT WORK (literally you cannot navigate months) in Firefox. This is specifically because of the way Firefox handles button elements. I don't know that we'll want to keep this forever but this div with a button role addresses the issue and still satisfies accessibility criteria. This is worth revisiting (and as I mentioned there still exists an issue with there being no visible outline) but we need to fix this right now.

Reply:
@backwardok I think I actually do want to keep the css:

Reply:
border, padding, and margin don't need to be set on `div`s because they don't have those things set by default